contributing/release-schedule.md

```diff
@@ -1,84 +1,154 @@
 # Release Schedule
 
-- [Detailed schedule](#detailed-schedule)
-- [Determining if your change is in production](#determining-if-your-change-is-in-production)
-- [Cherry picks](#cherry-picks)
-  - [Cherry pick criteria](#cherry-pick-criteria)
-  - [Process for requesting a cherry pick](#process-for-requesting-a-cherry-pick)
-- [AMP Dev Channel](#amp-dev-channel)
-- [AMP Release Candidate (RC) Channel](#amp-release-candidate-rc-channel)
-- [Release cadence](#release-cadence)
-- [Release freezes](#release-freezes)
+- [Release Channels](#release-channels)
+  - [Nightly](#nightly)
+  - [Weekly](#weekly)
+  - [LTS](#lts)
+- [Determining if your change is in a release](#determining-if-your-change-is-in-a-release)
+- [Release Cadence](#release-cadence)
+  - [Detailed schedule](#detailed-schedule)
+- [Cherry-picks](#cherry-picks)
+  - [Process for requesting a cherry-pick](#process-for-requesting-a-cherry-pick)
+- [Opt-In Cookie](#opt-in-cookie)
+  - [AMP Dev Channel (Experimental)](#amp-dev-channel-experimental)
+  - [AMP Beta Channel (Beta)](#amp-beta-channel-beta)
+- [Release Freezes](#release-freezes)
 
 A new release of AMP is pushed to all AMP pages every week on Tuesday. **Once a change in AMP is merged into the master branch of the amphtml repository, it will typically take 1-2 weeks for the change be live for all users.**
 
-["Type: Release" GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) are used to track the status of current and past releases (from the initial cut to canary testing to production). Announcements about releases are made on the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
+## Release Channels
 
-## Detailed schedule
+The AMP runtime and extensions are provided through a variety of different *release channels*. Each channel serves a purpose for developers and for the AMP HTML Project itself. See the [release cadence section](#release-cadence) for a more detailed breakdown of how and when code from the [`ampproject/amphtml`](https://github.com/ampproject/amphtml) repository makes it into release builds.
 
-We try to stick to this schedule as closely as possible, though complications may cause delays. You can track the latest status about any release in the ["Type: Release" GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) and the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
+### Nightly
```

Comment:
This doc can't be merged until we switch over to the nightly process, which we are planning on starting either 2nd or 3rd week of January

Reply:
I'm going to pull the **nightly** bits into a secondary branch so it's ready to merge at that time.

Reply:
Pulled out everything here referring to nightly. Created https://github.com/rcebulko/amphtml/pull/1 as a starting point for @danielrozenberg whenever ready to launch nightly channel.